### PR TITLE
Add regulations and policy pages

### PR DESCRIPTION
**Pull request by upsu90**
- add Regulations, Cookies policy and Privacy Policy pages
- create ParagraphText component
- add some text to these pages
- change filenames and components extension

Screenshot 2020-11-22 212721

### CODE REVIEW

File: web-client/src/components/Navigation.jsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ function Navigation() {
             <Link to = '/contact'>
                 <li>Contact</li>
             </Link>
+
```
Comment (JustD4nTe) [inline web-client/src/components/Navigation.jsx]: Uhm... Enter?

**Review by JustD4nTe — APPROVED** [review]

File: web-client/src/components/Navigation.jsx (hunk 1 above)
Comment (upsu90) [reply web-client/src/components/Navigation.jsx]: done

**Comment by KamilDonda** [comment]
Why did You remove this file?

**Comment by upsu90** [comment]
> Why did You remove this file?

@JustD4nTe asked me to do this

**Review by KamilDonda — APPROVED** [review]
Good job

**Review by Kwolik — APPROVED** [review]
ok

**Pull request merged**